test/uk/gov/hmrc/alcoholdutyreturns/repositories/CacheRepositorySpec.scala

```diff
@@ -1,67 +0,0 @@
-/*
```

Comment:
Test was flaky, plus it's not a unit test if it's accessing Mongo. We have a repo test in 'it' already.